react/react1/week1/todo-list/src/App.js

```diff
@@ -0,0 +1,12 @@
+import './App.css';
+import { MainToDo } from './MainToDo';
+import { TodoData } from './TodoData';
+function App() {
+  return (
```

Comment:
Install the prettier vscode extension and format your code

Reply:
okay,Thanks :)